forest/daemon/src/daemon.rs

```diff
@@ -91,7 +91,7 @@ pub(super) async fn start(config: Config, detached: bool) -> anyhow::Result<Db>
     // Useful if others want to use this node to bootstrap from.
     info!("PeerId: {}", PeerId::from(net_keypair.public()));
 
-    let mut ks = if config.client.encrypt_keystore {
+    let mut ks = if config.client.encrypt_keystore && atty::is(atty::Stream::Stdin) {
```

Comment:
Should `encrypt_keystore` be used as default mode once forest is released? In this case, probably it's better to ask users to set an environment variable `FOREST_KEYSTORE_PHRASE` in noninteractive mode

Reply:
I don't think we should fallback to non encrypted keystore in any case cuz IMHO it's a key security feature that cannot be compromised

Reply:
This shouldn't be a big deal at the moment. We would like to move the keys into a separate process as outlined in the architecture proposal. 

That being said, it's a long journey to get there, and, as you nicely caught it, in the non-interactive mode, it is impossible to set the phrase ourselves. I'll change it to accommodate this. Thanks.

Reply:
So what's the conclusion? Is this PR on hold now?

Reply:
We can address the bug in this PR by merging this and addressing the encrypted-keystore feature for non-interactive users in a separate issue and a PR. Does it make sense to you @hanabi1224 @elmattic ?

Reply:
Alternatively, let me know what you prefer. The result will be the same. :)